doc/api.md

```diff
@@ -1140,11 +1140,50 @@ Successful operations return `Content-Type` header with `application/json` value
 
 _**Response payload**_
 
-A JSON object with a services field that contains an array of services that match the request. See the
+A JSON object with a configGroups or services field that contains an array of services that match the request. See the
 [config group datamodel](#service-group-datamodel) for more information.
 
 Example:
 
+```json
+{
+    "configGroups": [
+        {
+            "resource": "/deviceTest",
+            "apikey": "801230BJKL23Y9090DSFL123HJK09H324HV8732",
+            "type": "Light",
+            "trust": "8970A9078A803H3BL98PINEQRW8342HBAMS",
+            "cbHost": "http://orion:1026",
+            "commands": [{ "name": "wheel1", "type": "Wheel" }],
+            "attributes": [
+                {
+                    "name": "luminescence",
+                    "type": "Integer",
+                    "metadata": {
+                        "unitCode": { "type": "Text", "value": "CAL" }
+                    }
+                }
+            ],
+            "lazy": [{ "name": "status", "type": "Boolean" }]
+        },
+        {
+            "resource": "/deviceTest2",
+            "apikey": "A21323ASDG12312ASDN21LWQEJPO2J123",
+            "type": "Switch",
+            "trust": "8970A9078A803H3BL98PINEQRW8342HBAMS",
+            "cbHost": "http://orion:1026",
+            "commands": [{ "name": "on", "type": "order" }],
+            "attributes": [
+                {
+                    "name": "swithc_status",
+                    "type": "boolean"
+                }
+            ]
+        }
+    ]
+}
+```
+
```

Comment:
This should replace the existing example, not adding a new one.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11